cms/utils/log_operations.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+
+from django.contrib.admin.models import LogEntry, ADDITION, CHANGE, DELETION
+from django.contrib.contenttypes.models import ContentType
+from django.utils.translation import ugettext_lazy as _
+
+
+"""
+Logs are all collected here to keep the messages and contents consistent across all operations.
+May seem repetitive but allows each log to be unique for different circumstances.
+Method names make the log being created clear and concise
+
+Issues with current solution:
+    - A log entry isn't created when a page is created using the new page wizard. Again showing a flaw with testing via the API and not the actual fn's!!
```

Comment:
Not sure I follow what you mean by "Again showing a flaw with testing via the API and not the actual fn's!!" Do you mean that tests are passing even though this is not being logged in that case?